src/librustdoc/passes/private_items_doc_tests.rs

```diff
@@ -30,7 +30,7 @@ impl<'a, 'tcx> DocFolder for PrivateItemDocTestLinter<'a, 'tcx> {
         let cx = self.cx;
         let dox = item.attrs.collapsed_doc_value().unwrap_or_else(String::new);
```

Comment:
We should probably rename this pass and module.

Reply:
I'm not sure I follow - this pass still looks for doc-tests in private items, it just also looks for public items as well. Were you imagining something like `doc_test_lints.rs`?

Reply:
Renamed the file, moved `look_for_tests` to this file, and added a short doc-comment explaining that it runs two different lints.